scm/helper.go

```diff
@@ -46,8 +46,6 @@ const (
 
 	// TeachersTeam is the team with all teachers and teaching assistants of a course.
 	TeachersTeam = "allteachers"
```

Comment:
Should we also rename the team to just `teachers`?

Reply:
I think we should, but I'm not sure how it would affect demoting/promoting teachers in existing courses.